src/js/background.js

```diff
@@ -39,6 +39,7 @@ function Badger() {
   var self = this;
 
   self.webRTCAvailable = checkWebRTCBrowserSupport();
+  self.firstPartyDomainPotentiallyRequired = testCookiesFirstPartyDomain();
```

Comment:
We should probably call this function periodically in case the user changes the setting during a session.

Reply:
See my other reply; I think we care about whether `firstPartyDomain` is supported, not whether `firstPartyIsolate` is enabled.

Reply:
:+1: